app/components/post_list/post_list.js

```diff
@@ -169,7 +170,7 @@ export default class PostList extends PureComponent {
     handleContentSizeChange = (contentWidth, contentHeight) => {
         if (this.state.contentHeight !== contentHeight) {
             this.setState({contentHeight}, () => {
-                if (this.state.postListHeight && contentHeight < this.state.postListHeight && this.props.extraData) {
+                if (this.state.postListHeight && contentHeight < this.state.postListHeight && !this.props.extraData && contentHeight > 53) {
```

Comment:
The `53` is here again, but I can't decipher its meaning... same as previous comment.